build/config.cmake

```diff
@@ -131,7 +139,10 @@ IF (${ESCARGOT_HOST} STREQUAL "tizen_obs")
     SET (ESCARGOT_CXXFLAGS ${ESCARGOT_CXXFLAGS} ${DLOG_CFLAGS_OTHER})
 ENDIF()
 
-SET (ESCARGOT_DEFINITIONS ${ESCARGOT_DEFINITIONS} -DENABLE_COMPRESSIBLE_STRING)
+SET (ESCARGOT_DEFINITIONS ${ESCARGOT_DEFINITIONS} -DENABLE_COMPRESSIBLE_STRING -DLZ4_MEMORY_USAGE=16 -DLZ4_HEAPMODE=1)
+IF (${ESCARGOT_SMALL_CONFIG} STREQUAL "ON")
+    SET (ESCARGOT_DEFINITIONS ${ESCARGOT_DEFINITIONS} -DLZ4_MEMORY_USAGE=16 -DLZ4_HEAPMODE=1)
```

Comment:
`-DLZ4_MEMORY_USAGE=16 -DLZ4_HEAPMODE=1` options are already applied in default.
Should these options be enabled only for `SMALL_CONFIG` mode?

Reply:
you are right